artemis-image/examples/byoc__etc/broker.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version='1.0'?>
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+<configuration
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns="urn:activemq"
+        xsi:schemaLocation="urn:activemq:core schema/artemis-configuration.xsd">
+   <core xmlns="urn:activemq:core">
+
+      <!--
+       Configure an image in the traditional way.
+       Providing boilerplate xml can form the well known configuration for an organisational base image that can be further extended with properties
+      -->
```

Comment:
Sure...so at least note very clearly here, that this image wont behave like the 'regular broker' does, or the artemis-docker based images do, if they just supplied a file such as this example is using? I think most users will be surprised at some of the ways it will differ.

Reply:
but why would an image behave like a regular broker? The image needs to be made. It needs to be customised. The whole point of an image is to lock it down for some scenario. To that end, this project makes an empty broker image. 
The "intent" bit of the readme is intended to convey that. Surprise is good, this is different!

Reply:
https://github.com/apache/activemq-artemis/pull/4297/files#diff-3e9a042c763f10a330adde9100e40b834b67abd3392bce668676fa36132b243aR7